cmd/list.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright © 2019 Zee Ahmed <zee@simplyzee.dev>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+)
+
+const (
+	binaryListURL = "https://api.github.com/repos/kubernetes/kubernetes/releases?per_page=100"
+)
+
+var (
+	remote bool
+)
+
+type kubectlVersion struct {
+	Version version.Version
+}
+
+func (kc *kubectlVersion) UnmarshalJSON(b []byte) error {
+	aux := &struct {
+		TagName string `json:"tag_name"`
+	}{}
+
+	if err := json.Unmarshal(b, &aux); err != nil {
+		log.Fatal(err)
+	}
+	version, err := version.NewVersion(aux.TagName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	kc.Version = *version
+	return nil
+}
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List downloaded kubectl binaries",
```

Comment:
Could the description be updated to support listing available remote downloads are now combined with the list argument instead?